src/app/components/transactionSetting/editBtcFee.tsx

```diff
@@ -0,0 +1,421 @@
+      try {
+        setIsLoading();
+        setError('');
+
+        if (isRestoreFlow) {
+          const { fee: modifiedFee, selectedFeeRate } = await getBtcFeesForNonOrdinalBtcSend(
+            btcAddress,
+            nonOrdinalUtxos!,
+            ordinalsAddress,
+            network.type,
+            feeMode,
+            feeRateInput,
+          );
+          setFeeRateInput(selectedFeeRate!.toString());
+          setTotalFee(modifiedFee.toString());
+        } else if (btcRecipients && selectedAccount) {
+          const { fee: modifiedFee, selectedFeeRate } = await getBtcFees(
+            btcRecipients,
+            btcAddress,
+            btcClient,
+            network.type,
+            feeMode,
+            feeRateInput,
+          );
+          setFeeRateInput(selectedFeeRate!.toString());
+          setTotalFee(modifiedFee.toString());
+        }
+      } catch (err: any) {
+        if (Number(err) === ErrorCodes.InSufficientBalance) {
+          setError(t('TX_ERRORS.INSUFFICIENT_BALANCE'));
+        } else if (Number(err) === ErrorCodes.InSufficientBalanceWithTxFee) {
+          setError(t('TX_ERRORS.INSUFFICIENT_BALANCE_FEES'));
+        } else setError(err.toString());
+      } finally {
+        setIsNotLoading();
+      }
+    } else if (type === 'Ordinals' && btcRecipients && ordinalTxUtxo) {
+      try {
+        setIsLoading();
+        setError('');
+
+        const { fee: modifiedFee, selectedFeeRate } = await getBtcFeesForOrdinalSend(
+          btcRecipients[0].address,
+          ordinalTxUtxo,
+          btcAddress,
+          btcClient,
+          network.type,
+          ordinalsUtxos || [],
+          feeMode,
+          feeRateInput,
+        );
+        setFeeRateInput(selectedFeeRate!.toString());
+        setTotalFee(modifiedFee.toString());
+      } catch (err: any) {
+        if (Number(err) === ErrorCodes.InSufficientBalance) {
+          setError(t('TX_ERRORS.INSUFFICIENT_BALANCE'));
+        } else if (Number(err) === ErrorCodes.InSufficientBalanceWithTxFee) {
+          setError(t('TX_ERRORS.INSUFFICIENT_BALANCE_FEES'));
+        } else setError(err.toString());
+      } finally {
+        setIsNotLoading();
+      }
+    }
+  };
+
+  useEffect(() => {
+    if (feeRateInput) {
+      setFeeRate(feeRateInput);
+    }
+  }, [feeRateInput]);
+
+  useEffect(() => {
+    if (debouncedFeeRateInput) {
+      recalculateFees();
+    }
+  }, [debouncedFeeRateInput]);
+
+  function getFiatEquivalent() {
+    return getBtcFiatEquivalent(new BigNumber(totalFee), BigNumber(btcFiatRate));
+  }
+
+  const getFiatAmountString = (fiatAmount: BigNumber) => {
+    if (fiatAmount) {
+      if (fiatAmount.isLessThan(0.01)) {
+        return `<${currencySymbolMap[fiatCurrency]}0.01 ${fiatCurrency}`;
+      }
+      return (
+        <NumericFormat
+          value={fiatAmount.toFixed(2).toString()}
+          displayType="text"
+          thousandSeparator
+          prefix={`~ ${currencySymbolMap[fiatCurrency]} `}
```

Comment:
```suggestion
          prefix={`~ ${currencySymbolMap[fiatCurrency]}`}
```

We would like to have no space between the fiat currency symbol and the amount, according to our design